web3/contract.py

```diff
@@ -698,17 +698,17 @@ def _none_addr(datatype, data):
 )
 
 
-class ConciseMethod:
+class ConciseMethod(object):
     ALLOWED_MODIFIERS = set(['call', 'estimateGas', 'transact', 'buildTransaction'])
 
     def __init__(self, function, normalizers=None):
         self.__function = function
         self.__function._return_data_normalizers = normalizers
 
     def __call__(self, *args, **kwargs):
-        return self.__prepared_function(*args, **kwargs)
+        return self._prepared_function(*args, **kwargs)
 
-    def __prepared_function(self, *args, **kwargs):
+    def _prepared_function(self, *args, **kwargs):
```

Comment:
I'm kind of paranoid about causing name conflicts, and this exposes us to another one. Some options are:

1. Rename to `_web3py_prepared_function` which would be a diabolical thing to call a Solidity function :P
1. Extract to a utility function, removing it from the class entirely, and pass in `self.__function` as an argument.
2. Rename to a Solidity keyword that makes sense, although the only one I can think of is `function` which is confusingly close to `__function` and the classic contract's `functions`.

I have a slight preference for 1, mostly because 2 might be a big rewrite. I have a strong preference for 2 over 3 (but could be convinced if the perfect keyword comes along).

Reply:
I could create *another* method in this class called something like `_get_prepared_function(*args, **kwargs)`, that calls this directly. Trying to figure out why this function needs to be totally private versus protected. Or I can just completely go `super()._ConciseContract__prepared_function(*args, **kwargs)` to call it in the subclass. Obviously not preferred, but if the shoe fits...

Reply:
Figured out a workaround